### DISCUSSION TITLE
2022 还能买房吗？

### DISCUSSION THREAD

**Discussion by alexwjj** [WeNeedHome/SummaryOfLoanSuspension#242] [Q&A]
到了结婚的年纪，老百姓都想有个家，买？烂尾暴跌等？不买，一直租也不是不行，女朋友估计不愿意。。。

**Comment by MyJuice** [comment]
买二手房所见即所得

**Reply by menglingyu** [reply]
所以贵都不算问题了?

**Reply by seeker-yao** [reply]
贵就等等， 不要把自己的风险搞的太大了，就业不稳定。

**Reply by qq912276337** [reply]
什么经济形势还想着买房？6个钱包？ 杠杆30年？ 不怕失业？不怕跌没首付款？全款？

**Reply by TALK1949** [reply]
> 所以贵都不算问题了?

二手房现在应该比一手便宜，因为政府限价只能约束地产商，二手房房价该崩还是会崩。

**Reply by seeker-yao** [reply]
> > 所以贵都不算问题了?
> 
> 二手房现在应该比一手便宜，因为政府限价只能约束地产商，二手房房价该崩还是会崩。

可以控制二手房的成交量的，比如贷款限制，比如二手房的落户政策啥的。

**Comment by kiss5523** [comment]
买现房或者二手吧，我本来以为老老实实当个小韭菜被割，谁知道买了个烂尾的好几年了，地方Zf和开发商都是一伙的，天天是个扯皮的，真的累。40万丢里面拿不回来了。

**Reply by Myosotissylvatica36** [reply]
不建议贷款买房，现在房子都是负资产，没有什么作用

**Reply by qq912276337** [reply]
什么经济形势还想着买房？6个钱包？ 杠杆30年？ 不怕失业？不怕跌没首付款？全款？

**Reply by MrChen2016-08-27** [reply]
40W？如果一直烂下去，现在你还要考虑后面贷款人生。。就算停贷要考虑资产转移和工资发放怎么走现金的事情了

**Reply by chenhaizano** [reply]
> 不建议贷款买房，现在房子都是负资产，没有什么作用

确实，现在人均都上百万存款，不建议贷款

**Comment by yalo-ddaa** [comment]
跟女朋友说房价要下跌了（

**Reply by Gemily4040** [reply]
虽然短期不至于大降 现阶段 依然不建议买房

**Reply by MYWmiss** [reply]
你居然试图跟女人讲道理。。

**Reply by imhappyok** [reply]
确实很多女性对房子有执念，说不通

**Reply by Gongkee** [reply]
> 确实很多女性对房子有执念，说不通

说不通就换个女朋友

**Reply by Streaking-snail** [reply]
> 确实很多女性对房子有执念，说不通

女性对房子没有执念，只是对不用自己出资的资产有执念。不信你试试和女性说不买房了，条件是把全款买房的资金全部送她作为个人财产他答不答应

**Reply by CKwuliuqi** [reply]
想想她是对房子有执念，还是对房子背后所代表的价值观有执念？

**Comment by Go-loneliness-of-man** [comment]
建议换个有脑子的女朋友

**Reply by redapple2021** [reply]
墙内有几个是清醒带脑子的。建议还是单身吧，结婚真不知道现在的意义在哪，再有个小的吞金兽，压力不是一般的大

**Comment by imahollister** [comment]
看地方啊，一线房价太高，新一线（特指沿海地区的这几个）的现房或者地段不错的二手房还是可以的。

**Comment by haoliangwu** [comment]
别买期房就行了，不然很大概率碰上这 repo 统计的破事儿，只买现房或者二手。

贵这东西没啥办法，因为大家都贵，不是光你一个人贵，嫌贵或者买不起就只能等或者想办法多挣钱咯。

女朋友愿不愿意就不评论了，反正我对象和我一起十二年，现在也没买房，去年也领证了，也挺好的。

几年前我确实买不起房，现在经过自己的努力也能买的起了，但我也不想买了，因为个破房子浪费自己 20、30 年的青春，不值得。

家的含义每个人理解不同，按我和我对象的理解，有没有房子无所谓，我俩只要在一起，无论在哪，都是家。

**Reply by qq912276337** [reply]
你这个才是正常的。房子是增加幸福感，而不是绑架幸福感

**Reply by yuanlimeisu** [reply]
是啊，就算买的起房，贷款30年，未来三十年要这么搭进去了

**Reply by flysheep311** [reply]
你最后这句话说的好，可惜有些女的感受不了，有人才是家

**Reply by haoliangwu** [reply]
> 是啊，就算买的起房，贷款30年，未来三十年要这么搭进去了

对啊，能想明白这道理的人不多，人生能有几个 30 年，往少说，可能就 1 个，往多说也不会超过 3 个，最好的年龄都在还贷款，想想就悲哀。

**Reply by haoliangwu** [reply]
> 你最后这句话说的好，可惜有些女的感受不了，有人才是家

这话也是我女朋友和我说的，当时给我整挺感动，我之前一直努力的目标非常简单粗暴，就是赚钱买房，算是给她一个我所谓的家，直到后来她和我说这句话才算彻底想明白事儿了。

这东西也不能说其他女的就不好，只能说价值观不同，就不评价了，但至少要是能遇到一个和我女朋友价值观一样的女孩，就多珍惜吧

**Reply by lyf9528** [reply]
主要是... 很多人努力奋斗不止是为了自己

**Comment by wenqian-deng** [comment]
建议好好跟女朋友聊一聊。房地产走向拐点，以前的想法也该变一变了。

**Comment by radiateboy** [comment]
没有现房就不买，这样可以真正多去几个楼盘挑一挑倒地谁的质量好，环境好。 光看模型有P用，知道什么叫仅供参考吗？

**Comment by DDFighting** [comment]
评估自己是不是真刚需呗，不买房就要分手，必须买，这种才算刚需，能拖的都不算。
刚需这种情况房价就不仅仅是房子了，还包括了你刚需的原因，这种没办法，因为你买的不仅仅是房子了，活在当下，10年后房子变成现在十分之一，但咱也等不起啊， 错过的可能再也回不来了。
当然，能跟女友谈妥最好，但是谁能保证之后一定跌呢，就算跌了也不能保证你喜欢的房子会跌啊，一线城市，强二线的核心区域保值还是比较稳定的，至少目前还是比较稳定的，我们这就是周边都多少有跌，但是热门区域坚如磐石。
当然也要评估自己的财务状况，保证杠杆程度比较健康，现在加高杠杆可能风险比较大。

**Comment by lettermessenger** [comment]
狗都不买

**Comment by wjlp2333** [comment]
不要为国接盘，怀疑下一个2008都快到了，手头的钱拿好最重要

**Comment by C-Joey** [comment]
早买早享受 晚买享折扣 反正除非是一线城市核心地段的房子，其他的是不可能涨了。 我在2019年的时候看到的数据居民收入负债比已经超过日本经济泡沫破裂的水平，也就是说一旦经济下行，地产会加速向下。如果你不是资金特别充裕，并且年龄没到35岁，建议先不买。如果你女朋友还是硬要买，那我认为她没有风险意识。

**Comment by haoww** [comment]
钱多的话随便，反正我是穷逼买不起，也不会买

**Comment by zsmhub** [comment]
房市有风险，入市需谨慎！

**Comment by kennard520** [comment]
别买房，我买了，想换大一点的，才知道楼市太差了，根本卖不掉

**Comment by stevenwsg** [comment]
我也在犹豫中

**Comment by markKscode** [comment]
去国外买房啊，新加坡、日本大量华人前去买房。

**Comment by wnhhh** [comment]
全款买二手房就比较稳妥

**Reply by didilinkin** [reply]
正解，多大的饭量吃多大的锅，没必要 全家举债 拼命买个超出能力范围的。 无债一身轻

**Comment by clleo888** [comment]
上策 : 赶快润
中策 : 租房 / 住进卖不出去的烂尾豪宅
下策 : 买房

**Reply by didilinkin** [reply]
你就不能租个正常点的居民楼， 非要租 烂尾？

**Reply by Humphrey75432** [reply]
租烂尾楼，这都出得什么馊主意？

**Reply by CKwuliuqi** [reply]
人家的意思是，要么租房住，要么去免费住烂尾的别墅

**Comment by Humphrey75432** [comment]
这样说吧，我在沿海某二三线城市。两周前中介带我去看了一套楼龄快10年的二手房。当时报价220万谢绝还价。我表示没兴趣之后。最近中介又告诉我这套二手可以210万成交。而且这个小区的入住率算比较高的，又离我上班的地方近。我还是没考虑。
所以要换作是我，我的建议是能租房先租，这几年经济形势不太乐观，还是以存钱为主，尽量别把自己的现金流都砸进房子。买房容易脱手难。实在要买一定要多去踩盘对比。优先考虑二手和一手现楼，期房能躲多远就躲多远。

**Reply by CKwuliuqi** [reply]
千万别考虑，现在二手房市场在大规模出清，中介给你出210W，你算算他自己中间抽多少水，他卖的高拿的多，这肯定不是售房人的真实售价，现在情况就是越不买，他越急着出手，越晚买能省好几年的打拼

**Comment by JayChou55** [comment]
应该会跌 20-30%非一线 但就算是这个价格 依然不便宜

**Reply by JayChou55** [reply]
并且 后期会配套房产税。。。不会让你太轻松的
